doc/source/Getting_started/Quickcode.rst

```diff
@@ -0,0 +1,127 @@
+Qhick Code
+==========
+
+Documentation and issues
+------------------------
+Documentation for the latest stable release of PyAEDT is hosted at
+`PyAEDT documentation <https://aedt.docs.pyansys.com/version/stable/>`_.
+
+In the upper right corner of the documentation's title bar, there is an option
+for switching from viewing the documentation for the latest stable release
+to viewing the documentation for the development version or previously
+released versions.
+
+You can also view or download PyAEDT cheat sheets, which are one-page references
+providing syntax rules and commands for using the PyAEDT API and PyEDB API:
+
+- `View <https://cheatsheets.docs.pyansys.com/pyaedt_API_cheat_sheet.png>`_ or
+  `download <https://cheatsheets.docs.pyansys.com/pyaedt_API_cheat_sheet.pdf>`_ the
+  PyAEDT API cheat sheet.
+
+- `View <https://cheatsheets.docs.pyansys.com/pyedb_API_cheat_sheet.png>`_ or
+  `download <https://cheatsheets.docs.pyansys.com/pyedb_API_cheat_sheet.pdf>`_ the
+  PyAEDT API cheat sheet.
+
+
+On the `PyAEDT Issues <https://github.com/ansys/PyAEDT/issues>`_ page, you can
+create issues to report bugs and request new features. On the `PyAEDT Discussions
+<https://github.com/ansys/pyaedt/discussions>`_ page or the `Discussions <https://discuss.ansys.com/>`_
+page on the Ansys Developer portal, you can post questions, share ideas, and get community feedback.
+
+To reach the project support team, email `pyansys.core@ansys.com <pyansys.core@ansys.com>`_.
+
+
+Example Workflow
+================
```

Comment:
```suggestion
================
```
We need to follow the rst headers. In that way we can avoid the error https://dev.docs.pyansys.com/how-to/documenting.html#indicating-rst-titles